src/lib/modules/Patron/PatronBulkExtendLoans/actions.js

```diff
@@ -0,0 +1,59 @@
+import { loanApi } from '@api/loans';
+import { searchReady } from '@api/utils';
+import {
+  sendErrorNotification,
+  sendSuccessNotification,
+} from '@components/Notifications';
+import {
+  fetchPatronCurrentLoans,
+  IS_LOADING as currentLoansIsLoading,
+} from '../PatronCurrentLoans/actions';
+import _isEmpty from 'lodash/isEmpty';
+
+export const IS_LOADING = 'bulkLoanExtend/IS_LOADING';
+export const SUCCESS = 'bulkLoanExtend/SUCCESS';
+export const HAS_ERROR = 'bulkLoanExtend/HAS_ERROR';
+
+export const bulkLoanExtension = (patronPid) => {
+  return async (dispatch) => {
+    dispatch({
+      type: IS_LOADING,
+    });
+    try {
+      const response = await loanApi.bulkExtendLoans(patronPid);
+      console.log(response);
+      if (!_isEmpty(response.data.extended_loans)) {
+        let message = `${response.data.extended_loans.length} 
+            loan(s) have been successfully extended.`;
+        const notExtendedMessage = ` ${response.data.not_extended_loans.length} 
```

Comment:
can `not_extended_loans` be null or something else? `extended_loans` was checked with an **if** with `isEmpty`, but not this one.